services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/component/DigitalGreenCertificateStructureProvider.java

```diff
@@ -63,20 +63,14 @@ public DigitalGreenCertificateStructureProvider(DistributionServiceConfig distri
    * Business rules Cbor encoded structures.
    */
   public DirectoryOnDisk getDigitalGreenCertificates() {
-    try {
-      return constructArchiveToPublish(distributionServiceConfig.getDigitalGreenCertificate());
-    } catch (UnableToLoadFileException e) {
-      logger.error("We don't generate a value-sets file and this shouldn't override existing ones.", e);
-      return new DirectoryOnDisk(EMPTY_STRING);
-    }
+    return constructArchiveToPublish(distributionServiceConfig.getDigitalGreenCertificate());
   }
 
-  private DirectoryOnDisk constructArchiveToPublish(DigitalGreenCertificate dgcConfig)
-      throws UnableToLoadFileException {
+  private DirectoryOnDisk constructArchiveToPublish(DigitalGreenCertificate dgcConfig) {
     DirectoryOnDisk dgcDirectory = new DirectoryOnDisk(dgcConfig.getDgcDirectory());
 
-    for (String currentLanguage: dgcConfig.getSupportedLanguages()) {
-      try {
+    try {
+      for (String currentLanguage : dgcConfig.getSupportedLanguages()) {
```

Comment:
Not sure but isnt this exactly what an indexingDirectory is used for? Providing indexing (in this case languages) and writing the same logic for each index and add its result to the directory? Maybe we can have a call about that